README.md

```diff
@@ -1,17 +1,29 @@
 ![StackMaster](/logo.png?raw=true)
 
-StackMaster is a sure-footed way of creating, updating and keeping track of
-Amazon (AWS) CloudFormation stacks.
-
-- See the changes you are making to a stack before you apply them
-- Connect stacks
-- Keep secrets secret
-- Customise stacks in different environments
-- Apply descriptive labels to regions
-
-StackMaster provides an easy command line interface to managing CloudFormation
-stacks defined with templates specified in either the
-[SparkleFormation](http://www.sparkleformation.io) DSL or standard JSON format.
+StackMaster is a CLI tool to manage CloudFormation stacks, with the following features:
+
+- Synchronous visibility into stack updates. See exactly what is changing and
+  what will happen before agreeing to apply a change.
+- Dynamic paramter resolvers.
+- Template compiler support for YAML and [SparkleFormation](http://www.sparkleformation.io).
+
+Stack updates can cause a lot of damage if applied blindly. StackMaster helps
+with this by providing the operator with as much information about the proposed
+change as possible before asking for confirmation to continue. That information
+includes:
+
+- Template body and parameter diffs.
+- [Change
+  sets](https://aws.amazon.com/blogs/aws/new-change-sets-for-aws-cloudformation/)
+are displayed for review.
+- Once the diffs & change set have been reviewed, the change can be applied and
+  stack events monitored.
+- Stack events will be outputted until an end state is reached.
```

Comment:
Is outputted a word?  I would have used output, but then I don't english very well so please do ignore me if I'm wrong.